docs/source/release-notes/2020-3-2-Demeter.md

```diff
@@ -0,0 +1,223 @@
+* Application: Qt 5.15 fixes: Initialize flags with default constructor. (#791)
+* Application: Add color, ms timestamps, PID, and thread ID as possible logging settings. (#804)
+* UI: Hide 'Secure Transactions' section in Settings > Security. (#842)
+* Application: Clean and revamp 'Script Plugin Infrastructure' example plugin. (#847)
+* Build: Fix GHA building. (#872)
+
+## Domain Server
+
+The numbers at the end of each item are the PR numbers in the Project Athena [repo](https://github.com/kasenvr/project-athena/).
+
+#### General
+
+* Server: Append 'Vircadia' to the environment variable for the metadata exporter port. (#826)
+* UI/UX: Fix wrapping of "Enable Automatic Threading" in server setup wizard. (#844)
+* UI/UX: Fix 'Create new Domain ID' dialog. (#846) 
+
+## Metaverse Server & Dashboard (Codename Iamus)
+
+### [Iamus](https://github.com/kasenvr/iamus) Metaverse Server
+
+#### Version 2.3.3
+
+<ul>
+<li><a href="http://github.com/kasenvr/Iamus/commit/fe0518f44cbbab7e87a68c8e2bf44a8149cf9262">view &bull;</a> Update Changelog.md</li> 
+<li><a href="http://github.com/kasenvr/Iamus/commit/f1299154d43fe10687de6bc8ee79460fa4e24e53">view &bull;</a> Add proper handling of SIGTERM and SIGINT     Shutdown server, finish requests, and then cleanly exit.</li> 
+<li><a href="http://github.com/kasenvr/Iamus/commit/6a8c2b0fab15558ef684555ff35843d030dcf718">view &bull;</a> Add domain.sponsorAccountId to Places information returned by /api/v1/places/... Update documentation.</li> 
+<li><a href="http://github.com/kasenvr/Iamus/commit/fc4f64670e5836f732b8cef1f5a5a477d3aea76f">view &bull;</a> Fix problem with changing username: update AccountEntity setPermissions. Should fix vircadia/project-iamus-dashboard#21</li> 
+<li><a href="http://github.com/kasenvr/Iamus/commit/8cf02628b3c8055d0e84655fa2e65951bd2a6a4b">view &bull;</a> Bump version to 2.3.3</li> 
+</ul>
+
+#### Version 2.3.2
+
+<ul>
+<li><a href="http://github.com/kasenvr/Iamus/commit/1a137f8f7b2722234ef5d3a5fd110d1007a502fd">view &bull;</a> Fix sorting of versions in genChangeLog.sh Update Changelog.md</li> 
+<li><a href="http://github.com/kasenvr/Iamus/commit/4654f30bb6af0cb89ac0a4c7fcf9f640576876f3">view &bull;</a> Return more useful error message when finding a domain name contains     non-allowed characters. Some non-functional formatting and code cleanup.</li> 
+<li><a href="http://github.com/kasenvr/Iamus/commit/729cdd156c95d514751a7663bafd6b584c725d49">view &bull;</a> Fix line endings to Linux style</li> 
+<li><a href="http://github.com/kasenvr/Iamus/commit/0511c01854bebf75a46a7ad79dbf243f194aa06d">view &bull;</a> Modify Entity field value validation to return reason for any validation failure.     Rather than just returning 'false', the validator returns a structure with         an optional reason for the validation failure. Modify the callers of Entity field setting to return error. Closes #48 Fix AccountEntity.Availability and AccountEntity.Roles to validate values correctly.</li> 
```

Comment:
```suggestion
<li><a href="http://github.com/kasenvr/Iamus/commit/0511c01854bebf75a46a7ad79dbf243f194aa06d">view &bull;</a> Modify Entity field value validation to return reason for any validation failure. Rather than just returning 'false', the validator returns a structure with an optional reason for the validation failure. Modify the callers of Entity field setting to return error. Closes #48 Fix AccountEntity.Availability and AccountEntity.Roles to validate values correctly.</li>
```